packages/bindings/package.json

```diff
@@ -47,7 +54,7 @@
     "@solarwinds-apm/bindings-linux-x64-musl": "workspace:*"
   },
   "engines": {
-    "node": ">=16"
+    "node": ">=16.13.0"
```

Comment:
Node 16.13 is the first Node 16 LTS and it properly supports ESM. Every previous 16.x release has been unsupported for years.